packages/terra-application-header-layout/docs/README.md

```diff
@@ -7,6 +7,19 @@ This component renders an application header layout. To be used with a terra-lay
 - Install with [npmjs](https://www.npmjs.com):
   - `npm install terra-application-header-layout`
 
+## Features
+
+### Skip to Content Button
+
+To improve accessibility of our UI for users using a keyboard to navigate, we have implement a 'Skip to Content' button that focuses on the main content node.
```

Comment:
I remember @neilpfeiffer mentioning long ago that it does help navigation by assistive technologies like screen-readers as well.